src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2129,9 +2130,11 @@ public static Map<String,String> getInvertiblePinMap(Series series, Unisim unisi
 	}
 
 	/**
-	 * Copies the logic and implementation of a set of cells from one design to another.
+	 * Copies the logic and implementation of a set of cells from one design to another.  This will
+	 * replace the destination logical cell instances with those of the source design. 
 	 * @param src The source design (with partial or full implementation)
-	 * @param dest The destination design (with matching cell instance interfaces) 
+	 * @param dest The destination design (with matching cell instance interfaces). If targeting
```

Comment:
Wait. Are you sure it's this param that is expected to be empty (and not the `srcToDestInstNames` param)

Reply:
Sure, moved it to the other param.